_includes/head.html

```diff
@@ -10,6 +10,7 @@
 <meta content="" name="description">
 <meta content="width=device-width, initial-scale=1" name="viewport">
 <!-- Place favicon.ico and apple-touch-icon.png in the root directory -->
+<link rel="shortcut icon" type="image/png" href="img/cyberg/favicon.png">
```

Comment:
shouldn't we put the favicon in the root directory?
>Place favicon.ico and apple-touch-icon.png in the root directory

Reply:
done